src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineStandardStaging.java

```diff
@@ -80,6 +83,13 @@ public void stageStandard(StageStandardConfiguration config) throws AppEngineExc
 
       cloudSdk.runAppCfgCommand(arguments);
 
+      //TODO : Move this fix up the chain (appcfg)
+      if (config.getRuntime() != null && config.getRuntime().equals("java")) {
```

Comment:
This continues to confuse me. I thought App Engine Standard Java did not use app.yaml?